code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -2383,7 +2383,7 @@
 		name = "Graphene"
 		id = "graphene"
 		result = "graphene"
-		required_reagents = list("fuel" = 4, "iron" = 1, "silicon_dioxide" = 1)
+		required_reagents = list("fuel" = 4, "iron" = 1, "silicon" = 1)
```

Comment:
Would like a comment to the effect that silicon is used to reduce complexity. The original recipe was supposed to be an abstraction of an existing process.